Exceptions/DocExceptions/DocDoesntExist.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+[Serializable]
+public class DocDoesntExist : Exception
```

Comment:
When declaring Exception class, make sure to include the "Exception" suffix, so in this case it would be `DocDoesntExistException`